README.md

```diff
@@ -11,24 +11,66 @@ This library is inspired by the [PHP library PDFMerger](https://github.com/myoky
 ## Code sample
 
 ```javascript
-const PDFMerger = require('pdf-merger-js');
+const PDFMerger = require("pdf-merger-js");
```

Comment:
This was propably done by your editor, but it is just good practice in open source to not change coding style of other people within one pull-request. Can you please mage sure your editor does not touch other lines of code, that do not "need  any change. Thanks!

Reply:
Good catch. I had prettier enabled globally by mistake. I'll go through and fix these. Thanks for the review.